packages/js/assets/user-feedback-form.js

```diff
@@ -0,0 +1,122 @@
+(function (window, document) {
+  // eslint-disable-next-line quotes
+  const TEMPLATE = `$$TEMPLATE$$`
+  const ENDPOINT = 'https://api.honeybadger.io/v1/feedback'
+  const OPTIONS_KEY = 'honeybadgerUserFeedbackOptions'
+
+  const HoneybadgerUserFeedbackForm = function () {};
+  HoneybadgerUserFeedbackForm.prototype.build = function () {
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const self = this
+    self.element = document.createElement('div')
+    self.element.id = 'honeybadger-feedback-wrapper'
+    self.element.innerHTML = TEMPLATE
+    self.element.onclick = function (e) {
+      if (e.target !== self.element) return;
+      self.close();
+    }
+
+    document.body.appendChild(self.element)
+
+    self.form = self.element.getElementsByTagName('form')[0]
+    self.form.addEventListener('submit', function (e) {
+      e.preventDefault()
+      self.submit()
+    })
+
+    const cancelButton = document.getElementById('honeybadger-feedback-cancel')
+    cancelButton.onclick = (e) => {
+      e.preventDefault()
+      self.close()
+    }
+    const closeButton = document.getElementById('honeybadger-feedback-close')
+    closeButton.onclick = (e) => {
+      e.preventDefault()
+      self.close()
+    }
+
+    const formOptions = self.getOptions()
+    const { messages = {}, buttons = {}, labels = {} } = formOptions
+    for (let i in messages) {
```

Comment:
A little confusing to use `i` here since you're looping over the object's properties and `i` usually indicates an index of an array. 

Could use something like `key` or `messageType` maybe.